src/meds_torch/utils/utils.py

```diff
@@ -69,7 +71,7 @@ def wrap(cfg: DictConfig, **kwargs) -> tuple[dict[str, Any], dict[str, Any]]:
         # things to do if exception occurs
         except Exception as ex:
```

Comment:
Could add a `pragma: no cover` comment here to avoid the code cov error given this is just logging.